client/components/TestRenderer/AssertionsFieldset/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { Fieldset } from '..';
+import styled from '@emotion/styled';
+import supportJson from '../../../resources/support.json';
+
+const Label = styled.label`
+    display: block;
+
+    input {
+        margin-right: 0.25rem;
+    }
+`;
+
+const AssertionsFieldset = ({ assertions, commandIndex, assertionsHeader }) => {
+    // Handle case where build process didn't include assertionResponseQuestion
+    const normalizedHeader = useMemo(() => {
```

Comment:
The value from `support.json` wasn't being correctly included with test commands in `assertionsHeader.descriptionHeader`. I am not sure but my sense is that this isn't from changes in this PR but instead comes from the sequence of updates to the test data in current development. If this isn't the case, let me know. Either way this calculation handles the issue and it can become a tiny tech debt task after all of the v2 changes have landed.

Reply:
The renderer in this app has to be built on top of the individual [collected json files](https://github.com/w3c/aria-at/blob/e617ea452142f92446792ca19d299d9fa99d740a/tests/resources/aria-at-harness.mjs#L116:L130). Architecturally, that means the .collected.json for any of these tests have to include the relevant data. Looking into this, it's clear there is some extending that can happen here to support that.

Reply:
I've submitted https://github.com/w3c/aria-at/pull/1005 as a potential fix. Let me know your thoughts.

There's other potential tech debt I've noticed, that may come up during future v2 format support work, that I think we can handle as they arise.

Reply:
Looking again at how this is used, I'm also not against just explicitly depending on the `support.json` in this use case. It should always be up to date because of the import script.

Reply:
Let's leave as-is for now and I'll check out #1005 for sweeping this up later. Definitely would be nice to avoid the weirdness of looking for the undefined value here.

Reply:
Circling back around - It doesn't seem that `collectedText.supportJSON` is coming through with the changes in #1005. I can look more into this next week if we think it is important to handle now.